<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbox_Browser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>741</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>文件夹选择</string>
    </property>
    <widget class="QLineEdit" name="ledt_Browser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Browser">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbox_List">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>191</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>下拉列表选择</string>
    </property>
    <widget class="QComboBox" name="cbox_List">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>File-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File-4</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbox_MultiSelect">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>541</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>去冗余多选框</string>
    </property>
    <widget class="QToolButton" name="btn_Add">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>37</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_Del">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>37</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_Unselect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>211</width>
       <height>231</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <item>
      <property name="text">
       <string>项目1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>项目2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>项目4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>项目5</string>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="list_Selected">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <item>
      <property name="text">
       <string>项目3</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_Export">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成配置文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Import">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>导入配置文件</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
